superset/templates/superset/base.html

```diff
@@ -38,4 +38,7 @@
     {% with filename="preamble" %}
       {% include "superset/partials/_script_tag.html" %}
     {% endwith %}
+    {% with filename="login" %}
+      {% include "superset/partials/_script_tag.html" %}
+    {% endwith %}
```

Comment:
Can we combine this condition with `preamble` or it doesn't support any syntax to combine?

Reply:
superset/partials/_script_tag.html  file using  js_manifest(filename)  function and to load multiple files we can use common name  like 'entry'  but  for two distinct files we need to add them separately